libs/components/layout/testing/src/box/box-harness.filters.ts

```diff
@@ -0,0 +1,7 @@
+import { SkyHarnessFilters } from '@skyux/core/testing';
+
+/**
+ * A set of criteria that can be used to filter a list of SkyBoxHarness instances.
```

Comment:
looks like we're not consistent about this but I think SkyBoxHarness should have back ticks around it so it appears formatted in the docs